canonical-websites/www.ubuntu.com
# WIP: Build desktop/statistics/ page

# WIP

## Done

- Built desktop/statistics/ page according to [copydoc](https://docs.google.com/document/d/153OkfZHM_eTfAoUZF3f38VMtiFzaa2txB9B2lvs8xXA/edit)

## TODO

- [x] Build basic template #3895
- [x] Build bar charts with dummy data #3972
- [x] Build pie charts with dummy data #3966
- [x] Build progress bar charts with dummy data #3967
- [ ] Build area chart with dummy data
- [ ] Build map component with dummy data
- [ ] Add toggles for hardware/VM data on per-chart basis
- [ ] Implement design suggestions from @anasereijo
- [ ] Replace dummy data with real, live data

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/desktop/statistics/
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check that it matches the [copydoc](https://docs.google.com/document/d/153OkfZHM_eTfAoUZF3f38VMtiFzaa2txB9B2lvs8xXA/edit)
